### ISSUE TITLE
New content date is invalid

### ISSUE THREAD

**Issue by luckyrat** [netlify/netlify-cms#578]
**- Do you want to request a *feature* or report a *bug*?**
bug

**- What is the current behavior?**
Creating new collection item results in an ambiguous date being created in the YAML front matter. It should be an ISO date but is instead "date: 03/22/2010 7:06 PM"

**- If the current behavior is a bug, please provide the steps to reproduce.**

1. Create a collection with a field like {label: "Date", name: "date", widget: "datetime"}
2. Load the CMS and create a new item
3. Select a date in the datetime field
4. Save the new item

**- What is the expected behavior?**
An ISO date should be used so that Hugo can build the pages generated by the Netlify CMS.
E.g. "Date: 2010-22-03T19:06:00+00:00"

**- Please mention your node.js, and operating system version.**
https://unpkg.com/netlify-cms@~0.4/dist/cms.js
Chrome 60

**- Workaround**

Use a standard string widget instead and manually manage the ISO format dates. Not a big problem for a technical user but a typical CMS user may struggle more.

**Comment by erquhart** [comment]
Thanks for reporting!

**Comment by BlueChili** [comment]
My current workaround is adding a format property to the widget config.
`- { label: "Date", name: "date", widget: "datetime", format: "Y-MM-DDThh:mm:ssZ"}`

**Comment by rriemann** [comment]
@BlueChili , there is only `Y` and not `YYYY` ?

Maybe this is the problem I have. I tried it with:

``` yml
- {label: "Conference Start", name: "begin", widget: "date", format: "YYYY-MM-DD", required: true}
```

**Comment by erquhart** [comment]
This is addressed in 0.7.6 - `moment.defaultFormat` which is ISO, is now used if no format is provided.

Note: prior to 0.7.6, the formatting field wasn't actually working, but it seems setting any format value at all at least caused the default format to be used. At any rate, formatting now works correctly, also as of 0.7.6.

**Comment by dthreatt** [comment]
I am still getting this bug
